src/server/dflycmd.cc

```diff
@@ -413,14 +413,11 @@ void DflyCmd::TakeOver(CmdArgList args, ConnectionContext* cntx) {
   absl::Time start = absl::Now();
   AggregateStatus status;
 
+  sf_->CancelBlockingCommands();
+
   // We need to await for all dispatches to finish: Otherwise a transaction might be scheduled
   // after this function exits but before the actual shutdown.
-  sf_->CancelBlockingCommands();
```

Comment:
following up on the previous question - it's been done already, no?

Reply:
No, it's for takeover